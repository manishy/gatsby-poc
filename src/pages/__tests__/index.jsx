import React from "react";
import renderer from "react-test-renderer";
import { PureIndexPage as Index } from "../index";
beforeEach(() => {
  // do some common stuff here
});

const data = {
  allContentfulBook: {
    edges: [
      { node: { slug: "some_slug", name: "some_book", author: "some_author" } },
    ],
  },
};

describe("Index", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Index data={data} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
