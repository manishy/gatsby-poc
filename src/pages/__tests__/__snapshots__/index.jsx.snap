// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders correctly 1`] = `
<div>
  <div>
    <div>
      <div>
        <a
          href="/book/some_slug"
        >
          <div
            className="tc grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5"
          >
            <h3>
              some_book
            </h3>
            <p>
              some_author
            </p>
          </div>
        </a>
      </div>
    </div>
  </div>
</div>
`;
